packages/custom-elements-manifest-tools/README.md

```diff
@@ -0,0 +1,37 @@
+# cutsom-elements-manifest-tools
+
+A collection of utilities for working with [Custom Element Manifest](https://github.com/webcomponents/custom-elements-manifest) files.
```

Comment:
Why wouldn't this stuff be part of the `custom-elements-manifest` package? Want to keep that package really lightweight, I assume?

Reply:
And to keep from churning the version of that package when the tools change. Tool breaking changes and schema breaking changes should be different.